components/edit-collective/EditTransferWiseAccount.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from 'react-apollo';
+import { useFormik } from 'formik';
+import { defineMessages } from 'react-intl';
+
+import { GraphQLContext } from '../../lib/graphql/context';
+import { gqlV2, API_V2_CONTEXT } from '../../lib/graphql/helpers';
+
+import { P } from '../Text';
+import StyledButton from '../StyledButton';
+import StyledInputField from '../StyledInputField';
+import StyledInput from '../StyledInput';
+
+const createConnectedAccountMutation = gqlV2`
+  mutation createConnectedAccount($connectedAccount: ConnectedAccountCreateInput!, $account: AccountReferenceInput!) {
+    createConnectedAccount(connectedAccount: $connectedAccount, account: $account) {
+      id
+      settings
+      service
+      createdAt
+    }
+  }
+`;
+
+const deleteConnectedAccountMutation = gqlV2`
+  mutation deleteConnectedAccount($connectedAccount: ConnectedAccountReferenceInput!) {
+    deleteConnectedAccount(connectedAccount: $connectedAccount) {
+      id
+    }
+  }
+`;
+
+const mutationOptions = { context: API_V2_CONTEXT };
+
+const EditTransferWiseAccount = props => {
+  const messages = defineMessages({
+    'collective.connectedAccounts.transferwise.button': {
```

Comment:
**[:ice_cream: Details & personal preferences]** I personally prefer to use simple keys for defined message objects, something like:

```suggestion
    button: {
```

That way you can simply do `formatMessage(messages.button)` to translate the string.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Nice, I'll start doing that.